### DISCUSSION TITLE
FYI: Windows 10 + WinPcap not working for me -> use Npcap instead if you have problems

### DISCUSSION THREAD

**Discussion by haja00011** [libpnet/libpnet#587] [General]
Hello,

just if someone has a similar problem. I use Windows 10 and tried to run _arp_packet_ example with _libpnet version 0.29.0_. 
I installed _WinPcap 4.1.3_ copied Packet.lib to correct place. It always hangs forever in:  `receiver.next().unwrap();`.
I checked with Wireshark, packet was sent and answered correctly.

I deinstalled WinPcap and installed _Ncap 1.71_ in WinPcap compatible mode there it works fine.

No idea why or if there is a workaround, just to let you know if you have a similar problem. (Don't forget to copy Packet.lib also for Ncap)

Btw: WinPcap is not in development anymore. Maybe we should always use Ncap.

Best regards
Jan

**Comment by Akmot9** [comment]
what do you think about the issue i opened about creating a documentation for windows user ? : https://github.com/libpnet/libpnet/issues/646
